script/deployParameters/bsc.json

```diff
@@ -1,20 +1,19 @@
 {
-    "permit2": "0x000000000022D473030F116dDEE9F6B43aC78BA3",
-    "weth9": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
-    "seaport": "0x00000000006c3852cbEf3e08E8dF289169EdE581",
-    "nftxZap": "0x0000000000000000000000000000000000000000",
-    "x2y2": "0x0000000000000000000000000000000000000000",
-    "foundation": "0x0000000000000000000000000000000000000000",
-    "sudoswap": "0x0000000000000000000000000000000000000000",
-    "nft20Zap": "0x0000000000000000000000000000000000000000",
-    "cryptopunks": "0x0000000000000000000000000000000000000000",
-    "looksRare": "0x0000000000000000000000000000000000000000",
-    "routerRewardsDistributor": "0x0000000000000000000000000000000000000000",
-    "looksRareRewardsDistributor": "0x0000000000000000000000000000000000000000",
-    "looksRareToken": "0x0000000000000000000000000000000000000000",
-    "v2Factory": "0x0000000000000000000000000000000000000000",
-    "v3Factory": "0x128Ce3A3D48f27CE35A3F810cF2cddD2f6879b13",
-    "pairInitCodeHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-    "poolInitCodeHash": "0xe34f199b19b2b4f47f68442619d555527d244f78a3297ea89325f843f87b8b54"
-  }
-  
\ No newline at end of file
+  "permit2": "0x000000000022D473030F116dDEE9F6B43aC78BA3",
+  "weth9": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
+  "seaport": "0x00000000006c3852cbEf3e08E8dF289169EdE581",
+  "nftxZap": "0x0000000000000000000000000000000000000000",
```

Comment:
ohhh we dont have seaport new stuff in here - please can you add seaportv1.4 and conduit here?